src/components/Timer/Timer.js

```diff
@@ -71,4 +81,24 @@ const Root = styled.div`
     color: pink;
 `;
 
+const displayTime = timeDiff => {
```

Comment:
That's definitely not very good name since it's ambiguous - one can think that it **displays** something (though it doesn't). I suggest changing it to something more explicit like `timeForDisplay` or `convertTimeToString`, or possibly `timerString` or you can get any more creative, it's up to you.